### ISSUE TITLE
Set Up Pods on Z Clusters and Execute Spyre Perf Suite on s390x

### ISSUE THREAD

**Issue by flop1971** [torch-spyre/torch-spyre#3118]
Placeholder for toolkit 581

**Comment by johnuman** [comment]
Pod successfully created on z cluster.

**Comment by johnuman** [comment]
Currently investigating executing Spyre Perf Suite on s390x

**Comment by johnuman** [comment]
I successfully completed a full end-to-end profiling workload on the s390x architecture using the benchmark script.  The test environment used the spyre-inference image, which included a preconfigured Python virtual environment with PyTorch 2.13, torch-spyre, and the spyre-perf-suite

I performed two workload runs:

Targeted Softmax workload
This command was used to reproduce the workload that had previously become stuck with high CPU usage:
spyre-perf-suite --op softmax --shape 1 512 4096 --stacks torch-spyre

Full default workload
This command was used to run the complete profiling workload:
spyre-perf-suite --default --stacks torch-spyre
